modules/govuk/manifests/apps/email_alert_service.pp

```diff
@@ -65,6 +70,20 @@
     process_regex  => '\/rake message_queues:unpublishing_consumer',
   }
 
+  govuk::procfile::worker { 'email-alert-service-subscriber-list-details-update-minor-consumer':
+    setenv_as      => $app_name,
+    enable_service => $enable_subscriber_list_update_queue_consumers,
+    process_type   => 'subscriber-list-details-update-minor-consumer',
+    process_regex  => '\/rake message_queues:subscriber_list_details_update_minor_consumer',
+  }
+
+  govuk::procfile::worker { 'email-alert-service-subscriber-list-details-update-major-consumer':
+    setenv_as      => $app_name,
+    enable_service => $enable_subscriber_list_update_queue_consumers,
+    process_type   => 'subscriber-list-details-update-minor-consumer',
```

Comment:
Now this `process_type` says `minor` but the rest of it is for the `major`

Reply:
yep gimme 1 sec, i'll hit the "re-review" button in a sec